app/main.py

```diff
@@ -14,4 +14,4 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    return sum(True for item in phrase.lower() if item == letter.lower())
```

Comment:
try to use a .count() method for phrase for better code style

Reply:
Done.